Profiles/Sassy/responses.yml

```diff
@@ -469,6 +469,16 @@ TimerReset:
 - Text: Timer reset
 TimerPaused:
 - Text: Timer paused
+LongSpeechResponse:
+- Text: ""
+- Text: "Wow, that sure took a long time to say"
+  Weight: 0.1
+- Text: "I better be getting paid overtime for how much I've been talking"
+  Weight: 0.1
+- Text: "If you want me to keep going, you can just ask me to sing a song for you."
```

Comment:
Won't this also get triggered on songs, though? I'd be a little weird for Tracker to say this after just finishing a song.